package.json

```diff
@@ -16,9 +16,11 @@
     "typescript",
     "react-native"
   ],
+  "peerDependencies": {
+    "react-native": "^0.41.2 || ^0.42.3"
```

Comment:
If you could change it that it is `>= 0.41.0 < 0.43.0` then I'll accept. I think that should cover everything.